src/modules/dashboard/components/ColonyHome/ColonyExtensions/ColonyExtensions.tsx

```diff
@@ -15,6 +15,10 @@ const MSG = defineMessages({
     id: 'dashboard.ColonyHome.ColonyExtensions.title',
     defaultMessage: 'Enabled extensions',
   },
+  loadingData: {
+    id: 'dashboard.ColonyHome.ColonyMembers.loadingData',
+    defaultMessage: 'Loading enabled extensions information...',
```

Comment:
I don't want to be picky but it just seems a bit too much for loading info. I saw that you just replicated what members loading shows but I would drop information from both. It's not that important.

Reply:
Agree, this should just be `Loading extensions data ...`